tests/v2/validation/provisioning/k3s/hardened_cluster_test.go

```diff
@@ -0,0 +1,135 @@
+//go:build (validation || sanity) && !infra.any && !infra.aks && !infra.eks && !infra.rke2k3s && !infra.gke && !infra.rke1 && !cluster.any && !cluster.custom && !cluster.nodedriver && !extended && !stress
+
+package k3s
+
+import (
+	"testing"
+
+	cis "github.com/rancher/rancher/tests/v2/validation/charts"
```

Comment:
There's a cross-import between validation packages here. Could you move benchmark helpers to the respective packages? These are great action examples that, if agreed upon, we can move after extensions vs actions.

Reply:
By moving to respective packages, do you mean in each cluster type? If so, that would be a bit of duplication since CIS benchmark is agnostic to cluster type. Also, it's not specific to provisioning, this can be used in other functional areas.

Reply:
Resolved